apps/svelte.dev/static/funding.json

```diff
@@ -0,0 +1,86 @@
+{
+	"$schema": "https://fundingjson.org/schema/v1.0.0/funding.schema.json",
+	"version": "v1.0.0",
+	"entity": {
+		"type": "organisation",
+		"role": "steward",
+		"name": "Svelte",
+		"description": "Web development for the rest of us",
+		"email": "",
+		"webpageUrl": {
+			"url": "https://svelte.dev"
+		}
+	},
+	"projects": [
+		{
+			"guid": "sveltejs-svelte",
+			"name": "Svelte",
+			"description": "Svelte is a UI framework that uses a compiler to let you write breathtakingly concise components that do minimal work in the browser, using languages you already know — HTML, CSS and JavaScript. It’s a love letter to web development.",
+			"webpageUrl": {
+				"url": "https://svelte.dev/docs/svelte"
+			},
+			"repositoryUrl": {
+				"url": "https://github.com/sveltejs/svelte",
+				"wellKnown": "https://github.com/sveltejs/svelte/blob/main/.well-known/funding-manifest-urls"
```

Comment:
Note that a file similar to https://github.com/sveltejs/svelte.dev/pull/1632/files#diff-dc6aa312408365efda3e35f463314bc3c7a2f37df864885c94ab21be318e8f6c needs to be added to the Svelte repository to provide provenance to the manifest.